CRM/Event/Cart/Form/Checkout/ParticipantsAndPrices.php

```diff
@@ -184,10 +206,38 @@ public function setDefaultValues() {
         $participant->contact_id = self::find_or_create_contact($this->getContactID());
       }
       $defaults += $form->setDefaultValues();
+      //Set price defaults if any
+      foreach ($this->cart->get_main_events_in_carts() as $event_in_cart) {
+        $price_set_id = CRM_Event_BAO_Event::usesPriceSet($event_in_cart->event_id);
+        if ($price_set_id) {
+          $price_sets = CRM_Price_BAO_PriceSet::getSetDetail($price_set_id, TRUE, TRUE);
+          $price_set  = $price_sets[$price_set_id];
+          foreach ($price_set['fields'] as $field) {
+            $options = CRM_Utils_Array::value('options', $field);
+            if (!is_array($options)) {
+              continue;
+            }
+            $field_name = "event_{$event_id}_price_{$field['id']}";
```

Comment:
To find the jenkins report you click on the red checkbox & it takes you there.

I fixed some yesterday though & I think we are green.

Looks like we just missed the cut-off timing for 4.7.9 - but if you confirm this is now good to go in your opinion I'll merge now for 4.7.10